### PR TITLE
Add jsonapi-serializer-cr

### PR DESCRIPTION
**Pull request by andersondanilo**

### CODE REVIEW

File: implementations/index.md [hunk 1]
```diff
@@ -315,6 +315,10 @@ includes related resources.
 
 * [json_api_server](https://pub.dartlang.org/packages/json_api_server) is a JSON:API server running on Dart VM.
 
+### <a href="#server-libraries-dart" id="server-libraries-dart" class="headerlink"></a> Crystal
```
Comment (gabesullice) [inline implementations/index.md]: I think you forgot to update the `href` and `id` attributes.

**Review by gabesullice — CHANGES_REQUESTED** [review]
Thanks! Just one small thing.

File: implementations/index.md (hunk 1 above)
Comment (andersondanilo) [reply implementations/index.md]: ok, i fixed

**Comment by andersondanilo** [comment]
@gabesullice, changes made!

**Review by gabesullice — APPROVED** [review]

**Comment by gabesullice** [comment]
Thanks!

**Pull request merged**